bindings/python/examples/missing_polygon.py

```diff
@@ -0,0 +1,39 @@
+from pymanifold import CrossSection, FillRule
+
+
+def run():
+    # create cross-sections
+    cross_section = \
+        CrossSection(
+            [[
+                (-176.5416564941429, 62.267156291299955), (-177.11900294911175, 63.267144940124126),
+                (-177.11900294910944, 62.26716470718384), (-176.5416564941406, 62.26715569064245),
+                (-176.54165614735393, 61.26716470718384), (-175.9643096923828, 61.26715985212647),
+                (-175.96430934559382, 60.26716470718384), (-175.38696566496216, 60.26717407055541),
+                (-175.386962543836, 59.26716470718384), (-174.80960083007815, 59.26714979522481),
+                (-174.8096004832914, 58.26716470718384), (-174.23225470415798, 58.26715427684856),
+                (-175.01954376826416, 59.63078409350792), (-176.5416564941429, 62.267156291299955),
```

Comment:
Can we further simplify this test case? I'm curious if it reproduces with simple integer coordinates.